### PR TITLE
Updated UI with new features

### PR DESCRIPTION
**Pull request by SachinMittal1766**
@ExpressHermes I have updated the UI of webpage. pls check it once.

### CODE REVIEW

File: views/partials/footer.ejs [hunk 1]
```diff
@@ -1,13 +1,14 @@
 
 <div style="position: absolute; bottom: 0; width: 100%; text-align: center; " class="ui inverted footer segment">
     <div class="ui container">
-        Made with
+        Designed and coded with
```
Comment (ExpressHermes) [inline views/partials/footer.ejs:4]: remove the 'and coded' just make it "Designed with"

File: public/stylesheets/app.css [hunk 2]
```diff
@@ -19,10 +19,26 @@ ui.container {
 }
 
 .jokeTypeButton{
```
Comment (ExpressHermes) [inline public/stylesheets/app.css:21]: format CSS properly

File: views/partials/jokeTypes.ejs [hunk 3]
```diff
@@ -1,5 +1,11 @@
-
+<!--
```
Comment (ExpressHermes) [inline views/partials/jokeTypes.ejs:1]: remove the comments

File: public/stylesheets/app.css (hunk 2 above)
Comment (SachinMittal1766) [reply public/stylesheets/app.css:21]: Can you pls explain what do you mean by "format CSS"

**Review by ExpressHermes — APPROVED** [review]

**Pull request merged**